CodeGeneration/Sources/SyntaxSupport/ExprNodes.swift

```diff
@@ -738,9 +738,9 @@ public let EXPR_NODES: [Node] = [
         kind: .token(choices: [.token(tokenKind: "BackslashToken")])
       ),
       Child(
-        name: "RawStringDelimiter",
+        name: "Pounds",
```

Comment:
Honestly I think I prefer `RawStringDelimiter`. That's mostly because `pounds` just reads weirdly to me though.

Reply:
Discussed offline, with `pounds`, I think is easier to understand what this represents.